### PR TITLE
Add category to SearchParams

### PR DESCRIPTION
**Pull request by Diogomartf**
image

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -62,13 +62,14 @@ export default function HomePage() {
...
   const showClientUI = useShowClientUI();
   const [isPopoverOpen, setPopoverOpen] = useState(false);
-  const [category, setCategory] = useState<CategoryOptions>("");
 
   const searchParams =
     typeof window !== "undefined"
       ? new URLSearchParams(window.location.search)
       : new URLSearchParams("/");
 
+  const [category, setCategory] = useState(searchParams.get("c") || "");
```
Comment (ElRodrigote) [inline app/page.tsx:71]: Is there a reason for us to use `c` instead `category` as search param?

**Review by ElRodrigote — APPROVED** [review]

File: app/page.tsx (hunk 1 above)
Comment (Diogomartf) [reply app/page.tsx:71]: we are using only one letter on search params.

No big reason, I just found it less verbose, and for URLs usually I like them short. 

The trade off is you lose a bit of readability but I don't expect people to create the url themselves, and for the super curious once you learn it, it's easier to write.

**Review by berteotti — APPROVED** [review]
I think scrollbar can be smaller. Otherwise looks good. Also I think we have too many categories, at least in the same bar.

**Comment by Diogomartf** [comment]
> I think scrollbar can be smaller. Otherwise looks good. Also I think we have too many categories, at least in the same bar.


> I think scrollbar can be smaller. Otherwise looks good. Also I think we have too many categories, at least in the same bar.

We will tweak the categories in the future, this is also a "transition" component, till we do the proper design. But it gets the functionality going.

**Pull request merged**